### ISSUE TITLE
PLANFIKSPUNKTDK.MIDDELFEJL

### ISSUE THREAD

**Issue by Helle-Git** [Kortforsyningen/FIRE-DDL#100]
Denne værdi kan ikke findes på helt samme måde som for højdefikspunkter.
For højdefikspunkterne anvendes sz-værdien for den tilhørende DVR90-koordinat; men der findes ikke altid en DVR90-koordinat for disse.
Hvordan findes den rigtige middelværdi for planfikspunkterne?

**Comment by kbevers** [comment]
Brug istedet gennemsnitsværdien af sx- og sy-værdierne. Med det data der ligger i FIRE lige nu er de altid ens, så i praksis er det det samme som at kun bruge sx-værdien, men jeg vil ikke afvise at der i fremtiden kommer forskellige værdier i de to felter.

**Comment by Helle-Git** [comment]
Er det en tilfældighed at de værdier jeg finder for SX, SY og SZ ser ud til altid at være 35, 35, 105; når SRIDID er 2.  (Jeg ser pt. kun på 500 planfikspunkter).
Hvis der er en DVR90-koordinat benyttes data fra denne; eller benyttes koordinat-værdier for SRIDID=2, hvis denne ikke findes, benyttes koordinat-værdisæt for SRIDID=3.

**Comment by kbevers** [comment]
> Er det en tilfældighed at de værdier jeg finder for SX, SY og SZ ser ud til altid at være 35, 35, 105; når SRIDID er 2. (Jeg ser pt. kun på 500 planfikspunkter).

Nej, det tror jeg ikke. Det er uden tvivl usikkerheder der er blevet vurderet ud fra nogle givne omstændigheder og ikke beregnet som en standardafvigelse på flere gentagne målinger.

> Hvis der er en DVR90-koordinat benyttes data fra denne; eller benyttes koordinat-værdier for SRIDID=2, hvis denne ikke findes, benyttes koordinat-værdisæt for SRIDID=3.

Brug sx/sy fremgangsmåden til dem alle. Da det er planfikspunktstabellen giver det ikke meningen at bruge middelfejlen på en højdekoordinat.
